frontend/src/app/plot/plot.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```

Comment:
group imports together.

Reply:
Since this file is autogenerated can I just leave it be? Same with the other .spec.ts files

Reply:
Any auto-generated files should be left as is. But in general this is good advice if you have files that are not auto-generated.